e2e/chat.spec.ts

```diff
@@ -397,7 +397,7 @@ describe('chat', () => {
     await expect(historyEntry).toHaveText('Test Conversation Title');
   });
 
-  test.skip('conversation search functionality', async ({ page }) => {
+  test('conversation search functionality', async ({ page }) => {
```

Comment:
Verifies the functionality that worked on the old UI works again (with minor updates)